t/17-labels_carry_over.t

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2016 SUSE Linux GmbH
```

Comment:
Well, @lnussel initiated discussion with legal about that and response was:

> SUSE LLC is the owner of SUSE IP. You should use it everywhere. But
don't delete but rather append.

Then there was this https://github.com/os-autoinst/openQA/pull/441 (see outdated diff) and since then we started replacing all to `SUSE LLC`.